packages/editor/example/uischema-example/config.ts

```diff
@@ -47,3 +47,24 @@ export const modelMapping = {
 };
 
 export const detailSchemata = {};
+
+export const importedUISchemaModel = {
```

Comment:
Does this belong here, where is it used?

Reply:
To test if the uploaded ui schemata can be rendered or not but I will remove it and move to ui editor application